src/data/list/basic.lean

```diff
@@ -1941,6 +1941,50 @@ begin
     apply hl _ (list.mem_cons_self _ _) }
 end
 
+/-- Transform a `l : list α` folded via `foldr` across a motive that holds
```

Comment:
"Transform ... across a motive" is not a sentence that means anything to me without reading the theorem statement! But if that's really standard usage of those words, fine. :-)